.github/workflows/manual-deploy-dev-staging.yml

```diff
@@ -18,11 +18,98 @@ on:
 env:
   DEVOPS_CHANNEL_ID: C37M86Y8G #devops-deploys
   VETS_WEBSITE_CHANNEL_ID: C02V265VCGH # status-vets-website
+  BUILD_ENV_DEV: vagovdev
+  BUILD_ENV_STAGING: vagovstaging
 
 jobs:
+  build:
+    name: Build
```

Comment:
We should probably use a matrix for this job so that we can build both build types simultaneously if `deploy_environment` is set to `both`, otherwise we'd have to wait for the dev build to finish before the staging build can start. They both use the same steps, so this would also be removing redundancy.